test/ir_Fujitsu_test.cpp

```diff
@@ -0,0 +1,149 @@
+  EXPECT_EQ(FUJITSU_AC_SWING_BOTH, fujitsuACSender.getSwing());
+  EXPECT_EQ(FUJITSU_AC_CMD_STAY_ON, fujitsuACSender.getCmd());
+
+  irsend.reset();
+  irsend.sendFujitsuAC(fujitsuACSender.getRaw(), 16);
+  EXPECT_EQ(
+  "m3224s1574m448s367m448s367m448s1182m448s367m448s1182m448s367m448s367m448"
+  "s367m448s1182m448s1182m448s367m448s367m448s367m448s1182m448s1182m448s367"
+  "m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448s1182"
+  "m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s367"
+  "m448s367m448s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s1182m448s1182m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s367m448s367m448s1182m448s1182m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s1182m448"
+  "s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448"
+  "s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s8100",
+  irsend.outputStr());
+}
+TEST(TestSendFujitsuAC, GenerateShortMessage) {
+  IRFujitsuAC fujitsuACSender = IRFujitsuAC(4);
+  IRsendTest irsend(4);
+  fujitsuACSender.begin();
+  irsend.begin();
+
+  fujitsuACSender.off();
+
+  EXPECT_EQ(FUJITSU_AC_CMD_TURN_OFF, fujitsuACSender.getCmd());
+
+  irsend.reset();
+  irsend.sendFujitsuAC(fujitsuACSender.getRaw(), 8);
+  EXPECT_EQ(
+  "m3224s1574m448s367m448s367m448s1182m448s367m448s1182m448s367m448s367m448"
+  "s367m448s1182m448s1182m448s367m448s367m448s367m448s1182m448s1182m448s367"
+  "m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448s1182"
+  "m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s367"TODO
+  "m448s367m448s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s1182m448s1182m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s367m448s367m448s1182m448s1182m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s1182m448"
+  "s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448"
+  "s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s8100",
+  irsend.outputStr());
+}
```

Comment:
test/ir_Fujitsu_test.cpp:149:  Could not find a newline character at the end of the file.  [whitespace/ending_newline] [5]